user-registration-app/src/app/views/live-view/live-view.component.html

```diff
@@ -45,11 +45,11 @@
       </div>
       <div class="col-md-4 d-flex">
         <a class="btn waves-effect waves-light submit-btn" target="_blank"
-           href="https://join.slack.com/t/hackpsuspring2019/shared_invite/enQtNTU5OTc2OTQ3Mzc3LTQ3NzQ5YzBmYzYxMDg0NDVkZDY2OTBlYjcwOWYzODAzMzlkZjZjZTA1NWY1OTg4OWIyNTRiMzBmMGQ1YjZiYzE">Slack</a>
+           href="hhttps://join.slack.com/t/hackpsu-group/shared_invite/enQtNzc4NDAzNjE4NjYwLTRjNjcyMDQwNmUzMTBiNDQzZDI2OWRjNWJjMTkwMzMzOGY2MGE2MDY5MDY1NjliNzFjNDE1YjVmMTJjN2U2Mjk">Slack</a>
```

Comment:
https

Reply:
:( big sad

Reply:
The link also doesn't seem to be active anymore

Reply:
fixed